packages/js/manifests/wrap/src/formats/wrap.info/0.1.schema.json

```diff
@@ -22,7 +22,8 @@
       "type": "string",
       "enum": [
         "wasm",
-        "interface"
```

Comment:
These changes should technically be going into a new version of the schema. Since no breaking changes are being made (if I'm correct) the version would be `0.2`.

Since this work is fixing things that should have been done before origin release, we can slide these changes in unnoticed right now IMO, but this should be the last time we do something like this.